nih-sparc/sparc-app
# Documentation page #407wyb

# Description

The purpose of this PR is to add the basic layout and new hero component to the documentation pages. In addition, I moved some things to the new patterns that we are using, such as removing [showdown](https://www.npmjs.com/package/showdown) to parse Markdown, in favor of our mixin for [marked](https://www.npmjs.com/package/marked).

## Type of change


# How Has This Been Tested?

- Visit the Help page using the main navigation
- The homepage should list articles, and have a temporary menu on the left.
- Click on one of the articles. This should show all content for that page, rendered as HTML.

# Checklist:
